app/views/volunteers/_form.html.erb

```diff
@@ -0,0 +1,97 @@
+<div>
+    <%= form_for @volunteer do |f| %>
+        <div>
+            <h4>Basic Information</h4>
+        </div>
+            <div>
+                <div>
+                    <h5>Volunteer Photo:</h5><%= f.text_field :picture, placeholder: 'Upload a profile picture.' %>
+                </div>
+                <div>
+                    <h5>First Name:</h5><%= f.text_field :first_name %>
+                </div>
+                <div>
+                    <h5>Last Name:</h5><%= f.text_field :last_name %>
+                </div>
+                <div>
+                    <h5>Email:</h5><%= f.text_field :email %>
+                </div>            
+                <div>
+                    <h5>Phone:</h5><%= f.telephone_field :mobile_phone %>
+                </div>            
+                <div>
+                    <h5>City:</h5><%= f.text_field :city %>
+                </div>            
+                <div>
+                    <h5>State:</h5><%= f.text_field :state %>
+                </div>            
+            </div>
+        <div>
+            <h4>Personal Information</h4>
+        </div>
+                <div>
+                    <h5>Birthdate:</h5><%= f.date_field :birthdate %>
+                </div>
+                <div>
+                    <h5>Gender:</h5><p>To which gender identity do you most identify with?</p>
+                    <%= f.select(:gender, options_for_select([
+                        ['Female', 'Female'],
+                        ['Male', 'Male'],
+                        ['Other', 'Other'],
+                        ['Prefer Not to Answer', 'Prefer Not to Answer']])) %>
```

Comment:
I think that you should be able to flatten this out more:

```suggestion
                    <%= f.select(:gender, options_for_select(['Female', 'Male', 'Other', 'Prefer Not to Answer'])) %>
```

Test that out to see if it works.